src/main/java/org/opentripplanner/standalone/config/sandbox/TransmodelAPIConfig.java

```diff
@@ -15,8 +15,9 @@ public class TransmodelAPIConfig implements TransmodelAPIParameters {
 
   private final boolean hideFeedId;
   private final Collection<String> tracingHeaderTags;
+  private final int maxNumberOfResultFields;
 
-  public TransmodelAPIConfig(String parameterName, NodeAdapter root) {
+  public TransmodelAPIConfig(NodeAdapter root) {
     var c = root
       .of("transmodelApi")
```

Comment:
The `parameterName` should be used here (my fault?) instead of being removed. Think of this as defining a type - you give the type a name and content, but the name of the instance is defined where it is used. It allow for reuse as well - not applicable here.